install/install.go

```diff
@@ -1533,3 +1596,15 @@ func (k *K8sInstaller) Install(ctx context.Context) error {
 
 	return nil
 }
+
+func (k *K8sInstaller) pushRollbackStep(step rollbackStep) {
+	k.rollbackSteps = append(k.rollbackSteps, step)
+}
+
+func (k *K8sInstaller) RollbackInstallation(ctx context.Context) {
+	k.Log("↩️ Rolling back installation...")
+
+	for _, r := range k.rollbackSteps {
+		r(ctx)
+	}
```

Comment:
It would be good to roll back in the reverse order, just in case there happen to be some dependencies between the created resources. I don't know if deleting in the order of creation would be a problem, but in general deleting resources in the reverse order of creation is a good practice.

Reply:
Makes sense :+1: I changed `pushRollbackStep` to prepend the step to the slice of steps